tests/framework/pvc.go

```diff
@@ -340,7 +340,7 @@ func (f *Framework) VerifyPermissions(namespace *k8sv1.Namespace, pvc *k8sv1.Per
 	return f.verifyInPod(namespace, pvc, cmd, func(output, stderr string) (bool, error) {
 		fmt.Fprintf(ginkgo.GinkgoWriter, "INFO: permissions of disk.img: %s\n", output)
 
-		return strings.Compare(output, "-rw-rw----.") == 0, nil
+		return strings.Compare(output, "-rw-rw----") == 0 || strings.Compare(output, "-rw-rw----.") == 0, nil
```

Comment:
So reading about it I see this is about a selinux context:
https://superuser.com/questions/230559/what-does-the-dot-mean-at-the-end-of-rw-r-r-how-do-you-set-it-with-chmod
But I wonder how we've never hit this before..

Reply:
Maybe because the host system is ubuntu.

Reply:
Hmm maybe strings.Contains is cleaner in this case? or use `stat` for numerical representation?

Reply:
> Hmm maybe strings.Contains is cleaner in this case? or use `stat` for numerical representation?

I think strings.Contains is good